scripts/polkadot-launch/composable.nix

```diff
@@ -0,0 +1,8 @@
+# definition of parachain
+# TODO: replace with zombienet
```

Comment:
Do we have a tracking issue?

Reply:
zombinet was told to be official runner and polkalaunch in maintain mode. for last months https://github.com/paritytech/zombienet/ got nice upgrades .  it is just notification that investing intp polkalaunch may be dead end so.

Reply:
but it is not yet clear if investing into zombienet is good. it allows tests and hooks which is great. runs on k8s/hative/docker right away. so kind of it makes many people happy than without.